### ISSUE TITLE
Working?

### ISSUE THREAD

**Issue by agent-r** [jcbhmr/fetch.ino#6]
Hi there,

this project looks like it is exactly what I am looking for, right now: An async https get requests library for eps32. Unfortunately this library seems to be "under construction" at the moment, right? (i can not get it working and i see the label in the readme.md)

Are you working on it, these days? Do you see a chance to get it running in the next few weeks?

Thanks for your work!
Paul

**Comment by jcbhmr** [comment]
Yep, still working on it! If you're looking for a fetch() function right now, check out the upstream package https://github.com/instanceofMA/arduino-fetch ❤️

The reason I'm doing this fork thing is due to these. I _especially_ desire a `.json()` function for Arduino.
![image](https://user-images.githubusercontent.com/61068799/233799269-6d87cab3-5a18-4561-a7a1-f5417a6733a0.png)

😊 Thanks for giving me some kind words of encouragement and indicating that, yes, someone _actually cares_ and _actually wants_ to use this project! 🙏💕
